### PR TITLE
Make errors more consistent

### PR DESCRIPTION
**Pull request by conradgrobler**
Error messages and additional information throughout the codebase have been somewhat inconsistent. This PR tries to make it a bit more consistent, mostly around the following:

- Don't capitalize the first letter of an error message
- Don't end error messages with a `.`
- Use consistent forms of words (mostly `couldn't` over `could not` and `failed to`)

### CODE REVIEW

File: experimental/sev_guest/src/msr.rs [hunk 1]
```diff
@@ -248,7 +248,7 @@ impl TryFrom<u64> for PreferredGhcbGpaResponse {
     fn try_from(msr_value: u64) -> Result<Self, &'static str> {
         const PREFERRED_GPA_RESPONSE_INFO: u64 = 0x011;
         if msr_value & GHCB_INFO_MASK != PREFERRED_GPA_RESPONSE_INFO {
-            return Err("Value is not a valid preferred GHCP GPA response");
+            return Err("Vvlue is not a valid preferred GHCP GPA response");
```
Comment (tiziano88) [inline experimental/sev_guest/src/msr.rs:251]:
```suggestion
            return Err("value is not a valid preferred GHCP GPA response");
```

File: experimental/virtio/src/console/mod.rs [hunk 2]
```diff
@@ -148,38 +148,38 @@ where
...
-            .context("Couldn't initialize the PCI device.")?;
+            .map_err(|error| anyhow::anyhow!("virtio error: {:?}", error))
+            .context("couldn't initialize the PCI device")?;
         self.device
             .configure_queue(
                 RX_QUEUE_ID,
                 QUEUE_SIZE as u16,
                 translate(self.rx_queue.inner.get_desc_addr())
-                    .context("Failed to translate VirtAddr to PhysAddr")?,
+                    .context("failed to translate VirtAddr to PhysAddr")?,
```
Comment (tiziano88) [inline experimental/virtio/src/console/mod.rs:158]: Maybe use `couldn't` here too?

**Review by tiziano88 — APPROVED** [review]
Thanks for doing it! That was a lot of changes! I hope you managed to use search & replace for at least some of them

File: experimental/virtio/src/console/mod.rs (hunk 2 above)
Comment (conradgrobler) [reply experimental/virtio/src/console/mod.rs:158]: I was undecided about that, but will go ahead with it

File: experimental/sev_guest/src/msr.rs (hunk 1 above)
Comment (conradgrobler) [reply experimental/sev_guest/src/msr.rs]: Done, thanks

**Review by tiziano88 — APPROVED** [review]

**Pull request merged**